src/zodbbrowser/browser.py

```diff
@@ -31,6 +33,8 @@
 
 log = logging.getLogger("zodbbrowser")
 
+__version__ = pkg_resources.get_distribution('zodbbrowser').version
```

Comment:
Ah, I see it's to support zest.releaser.  Yeah, I'm convinced.

(Usually I'm wary of relying on pkg_resources for version fetching because I want my code to be runnable from a source checkout, without a separate install step. I consider `virtualenv . && bin/pip install -e .`  to be a *build* step rather than an install step, and I'm fine requiring that.)